sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -117,6 +118,12 @@ class SQLContext private[sql](
    *
    * @since 1.6.0
    */
+
+  /** Pre-Load built-in User Defined Types */
```

Comment:
This part can be removed too. I would say this is probably the right thing to do if the registry is public. However, we might need to have a major refactor on UDTs in 2.1. It would be better to keep the current implementation as simple as possible.